runners/flink/1.10/build.gradle

```diff
@@ -17,6 +17,13 @@
  */
 
 def basePath = '..'
+def jackson_annotations_version = "2.10.2"
+def flink_clients_version = "1.10.1"
+def jackson_databind_verisioon = "2.10.2"
+def jsr305_version = "3.0.2"
+def flink_annotations_version = "1.10.1"
+def flink_optimizer_version = "1.10.1"
+def checker_qual_version = "2.0.0"
```

Comment:
This version should match the same version as all the other modules unless there is a reason to diverge. It is already added to every Java module by `BeamModulePlugin`.

Reply:
Flink 1.8 needs :
def flink_clients_version = "1.8.3"
def flink_annotations_version = "1.8.3"
def flink_optimizer_version = "1.8.3"
Flink 1.9 needs: 
def flink_clients_version = "1.9.3"
def flink_annotations_version = "1.9.3"
def flink_optimizer_version = "1.9.3"
Same is true for flink 1.10 and 1.11.